### PR TITLE
configure dependabot

### PR DESCRIPTION
**Pull request by Sharqiewicz**
🦈 PR - in progress
---

Dependabot will check daily for new versions of dependencies but will only apply updates for security vulnerabilities. Non-security updates are ignored to reduce noise. The packages will be updated to the latest versions [?].

Pull requests created by Dependabot will be prefixed with “bot-chore” in their commit messages, limited to a maximum of 5 open PRs. Each PR will be automatically labeled with “dependencies” and “security,” assigned to @Sharqiewicz . All updates will be grouped into a single PR for easier management.

### CODE REVIEW

**Comment by Sharqiewicz** [comment]
@ebma @TorstenStueber ready for quick review

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    commit-message:
+      prefix: "bot-chore:"
+    open-pull-requests-limit: 10
+    ignore:
+      - dependency-name: "*"
```
Comment (TorstenStueber) [inline .github/dependabot.yml:11]: Doesn't this mean that we ignore all dependencies?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    security-updates: true
```
Comment (TorstenStueber) [inline .github/dependabot.yml:12]: Couldn't find this top level setting in the [dependabot docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file), sure that it is correct?

**Review by TorstenStueber — COMMENTED** [review]
The PR looks generally good, I have two comments, though.

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    commit-message:
+      prefix: "bot-chore:"
+    open-pull-requests-limit: 10
```
Comment (ebma) [inline .github/dependabot.yml:9]: Can we limit this to 5 instead? 10 is a lot 👀

**Review by ebma — APPROVED** [review]

**Pull request merged**